adapters/saml/as7-eap6/subsystem/pom.xml

```diff
@@ -140,11 +140,21 @@ projects that depend on this project.-->
                     <name>product</name>
                 </property>
             </activation>
+            <repositories>
+                <repository>
+                    <id>redhat-ga</id>
+                    <url>https://maven.repository.redhat.com/ga/</url>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </repository>
+            </repositories>
             <dependencies>
                 <!-- Downstream version 7.5.*.Final is type pom see KEYCLOAK-11527 -->
                 <dependency>
                     <groupId>org.jboss.as</groupId>
                     <artifactId>jboss-as-subsystem-test</artifactId>
+                    <version>7.5.22.Final-redhat-1</version>
```

Comment:
I don't like we're hardcoding version here. Shouldn't we at least move it to the root POM?

Reply:
Sure. I'll create property in the root pom.